airflow/www/fab_security/manager.py

```diff
@@ -63,6 +63,35 @@
 # Copyright 2013, Daniel Vaz Gaspar
 log = logging.getLogger(__name__)
 
+__lazy_imports = {
+    "AUTH_DB": "flask_appbuilder.const",
+    "AUTH_LDAP": "flask_appbuilder.const",
+    "LOGMSG_WAR_SEC_LOGIN_FAILED": "flask_appbuilder.const",
+}
+
+
+def __getattr__(name: str):
+    # PEP-562: Lazy loaded attributes on python modules
+    path = __lazy_imports.get(name)
+    if not path:
+        raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+
+    import warnings
+
+    warnings.warn(
```

Comment:
I'd also be open to not deprecating importing from this module. Thoughts @vincbeck?

Reply:
Yep, I am good with that

Reply:
yes